src/app/components/sidebar/sidebar.component.scss

```diff
@@ -0,0 +1,123 @@
+
+
+.sidenav {
+   
+    background:#a99892;
+    transition: all 1.5s ease;
+    z-index: 1;
```

Comment:
This color is present in _imports.scss as $navbar-color. Use _imports.scss colors.